examples/pytorch/mixhop/README.md

```diff
@@ -0,0 +1,99 @@
+# DGL Implementation of MixHop
```

Comment:
"DGL Implementation of MixHop" -> "DGL implementations of MixHop" for the consistency across the file.

Reply:
Done.